testSuite/constraints.go

```diff
@@ -0,0 +1,29 @@
+package testSuite
+
+import "math"
+
+func constraint1(x []float64, g func([]float64) float64) bool {
+	return (0.51-g(x))*(g(x)-0.5) >= 0
+}
+
+func constraint2(x []float64) bool {
+	return math.Sin(20*math.Pi*x[0])-0.5 >= 0
+}
+
+func constraint3(x, p, q, a, b []float64, k int, f1, f2 float64) bool {
```

Comment:
trenger kanskje ikke sende inn x siden vi allerede bruker den utregnede verdien for f1 og f2 (dette burde nevnes i omdel/experiment kapittelet)